packets.go

```diff
@@ -1149,7 +1149,7 @@ func (rows *binaryRows) readRow(dest []driver.Value) error {
 			continue
 
 		case fieldTypeFloat:
-			dest[i] = float64(math.Float32frombits(binary.LittleEndian.Uint32(data[pos : pos+4])))
+			dest[i] = float32(math.Float32frombits(binary.LittleEndian.Uint32(data[pos : pos+4])))
```

Comment:
Well, this change fixes the issue I was running into and the tests I added cover both cases and passed on Travis.

I added an issue to the Go repo so they will hopefully fix the documentation to include the full list of types that are *actually* allowed in returns from drivers.